.github/workflows/main.yml

```diff
@@ -4,6 +4,9 @@ on:
     branches: [ main, 'stable/*' ]
   pull_request:
     branches: [ main, 'stable/*' ]
+concurrency:
+  group: ${{ github.repository }}-${{ github.ref }}-${{ github.head_ref }}
```

Comment:
The github docs say `group: ${{ github.head_ref }}`
Is there a difference with the way you've written it here?

Reply:
Not in a practical sense. The difference is the concurrency group name with this is something like:

`Qiskit/qiskit-experiments-refs/pull/226/merge-ci-concurrency-cancellation`

while doing it as the github docs say the concurrency group would be:

`ci-concurrency-cancellation`

I opted for the longer string because it's less likely to have a name conflict with another PR. The way the github docs do it if 2 PRs have the same branch name they'll be treated as a single concurrency group and cancel each others job. Not sure how likely that is in practice, but it was simple enough to make the string more likely to be unique.